### PR TITLE
Add Test cases for modules and JsonAdaptedModule

### LINKED ISSUE #49: Write test cases for Module and its attributes

### PR DESCRIPTION
**Pull request by lkldev**
closes #49

### CODE REVIEW

File: src/test/data/TaModuleTest/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Files are downloaded from NUSMods for academic year 2021/2022, to update the list, you can download them and replace the files here. 
+
+The links for academic year 2021/2022 are:
+1. https://api.nusmods.com/v2/2021-2022/moduleList.json
+2. https://api.nusmods.com/v2/2021-2022/moduleInformation.json
+
+You should update the academic year accordingly to download the upddated moduleList and moduleInformation.
```
Comment (jxt00) [inline src/test/data/TaModuleTest/README.md:7]: `upddated` is spelled wrongly.

**Review by jxt00 — APPROVED** [review]
LGTM

File: src/test/data/TaModuleTest/README.md (hunk 1 above)
Comment (lkldev) [reply src/test/data/TaModuleTest/README.md]: resolved in latest commit

**Review by wxliong — APPROVED** [review]
LGTM! :)

**Pull request merged**